src/actions/referralsActions.js

```diff
@@ -191,6 +188,10 @@ export const sendReferralInvitationsAction = (invitationContacts: ReferralContac
         sentInvitationsCount: { count: updatedInvitationCount, date: currentDate },
       }));
     }));
+    if (unsentInvitations.length < invitations.length) {
+      navigate(REFERRAL_SENT);
+    }
+    dispatch(inviteErrorAction(errorMessage, unsentInvitations.length === invitations.length));
```

Comment:
You're dispatching an error even though all invitations are sent?

Reply:
Yeah, missed that one. Thanks! Fixed :)